etc/skydive.yml.default

```diff
@@ -68,6 +68,8 @@ analyzer:
     # from the beginning automatically
     # capture_gremlin: "G.V().has('Name', NE('lo'))"
     # capture_bpf: "port 80"
+    # Capture type: pcap [default], afpacket, ebpf, sflow, pcapsocket, ovsmirror, dpdk, ovssflow, or ovsnetflow
```

Comment:
nit: The colon makes the sentence look a bit like valid YAML. Can you please change it to :
`Specify the capture type. Can be either pcap (the default), afpacket, ebpf, sflow, pcapsocket, ovsmirror, dpdk, ovssflow, or ovsnetflow`

Reply:
I'll fix the comment sentence.

Regrading the default: for *startup* captures, `analyzer/server.go` line 128 specifically hard-coded:

    capture.Type = "pcap"

Or maybe this is overridden somewhere else?

Reply:
It seems you did set the default value in config/config.go in the first version of your PR, or I missed it.

I would prefer to remove the line in config/config.go and let Skydive decide the type of the capture (so that it works on Openvswitch bridges for instance)

Reply:
I did set the default in `config/config.go` in order to keep the behavior before the change, which hard-coded startup captures to be of type `pcap`.

I'll modify it so the default is an empty string, which will let Skydive decide the type of the capture per node.

Note that this will break existing behavior of startup captures which were always of type `pcap`; now such users will have to add this config to keep their existing behavior:

```
analyzer:
  startup:
    capture_type: pcap
```

(or the corresponding environment variable.)

Reply:
Fixed in 83b1d8c05114581a98550c37ec8bccd28decb0f7 (force-pushed).